packages/experimental/Badge/src/Badge.styling.ts

```diff
@@ -17,9 +17,9 @@ import { defaultBadgeTokens } from './BadgeTokens';
 import { defaultBadgeColorTokens } from './BadgeColorTokens';
 import { badgeFontTokens } from './BadgeFontTokens';
 
-export const coreBadgeStates: (keyof BadgeCoreTokens)[] = [...BadgeSizes, ...BadgeShapes];
+export const coreBadgeStates: (keyof BadgeCoreTokens)[] = [...BadgeSizes, ...BadgeShapes, 'textPadding'];
```

Comment:
I don't think it makes sense for "textPadding" to be a state.

Reply:
Will remove it. I was trying to fix a bug with padding. It didn't work for the Text component